### ISSUE TITLE
Consider moving sagemath gitlab org?

### ISSUE THREAD

**Issue by embray** [tscrim/coxeter#17]
Hi @tscrim , 

We've been putting some math packages for which maintenance has been largely taken over by the Sage community (including, similarly to this library, symmetrica: https://gitlab.com/sagemath/symmetrica)

Would you be okay if we set up a repo for this on gitlab and moved all the issues over there?  That will make it easier also to have a new release upstream for use in Sage as well as downstream packagers.  I could also take over reworking the build system if you want.  This is motivated in part by https://trac.sagemath.org/ticket/29150

**Comment by tscrim** [comment]
Yes, that would be fine. This has ended up being too low of a priority for me, and I would appreciate any help. Thank you.
